### ISSUE TITLE
UiFileupload not clearing on cancel

### ISSUE THREAD

**Issue by z1haze** [JosephusPaye/Keen-UI#363]
The file upload input should clear when the select file action is 'cancelled'. Currently it keeps the file in the input, so there's no way to actual remove that file if you choose not to upload a file.

**Comment by YamenSharaf** [comment]
I'm having this issue too. Any pointers?

**Comment by EmilMoe** [comment]
I'm clearing mine, after upload, with this piece of code:

~~~~
this.$refs.upload.hasSelection = false
this.$refs.upload.displayText = this.$refs.upload.label
~~~~

**Comment by JosephusPaye** [comment]
Added a new method `clear`, to clear the selected file. https://github.com/JosephusPaye/Keen-UI/commit/9d2bc88f573d396d710bba596d73bdd97c58df62

**Comment by JosephusPaye** [comment]
Released in [v1.1.0](https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.1.0).

**Comment by kilobyte2007** [comment]
Apparently the clear() method is not working.
